automatic/ultravnc/tools/chocolateyInstall.ps1

```diff
@@ -23,5 +23,9 @@ $packageArgs = @{
   validExitCodes= @(0,3010)
 }
 
+$pp = Get-PackageParameters
+if ($pp['LoadInf'] -and !(Test-Path $pp['LoadInf'])) { throw "$($pp['LoadInf']) is not a valid file" }
+$packageArgs['silentArgs'] += " /loadinf:$($pp['LoadInf'])"
+
```

Comment:
This will always add the ini file as parameter, please only update silent args when there is actually an ini-file provided.